dyno/panda/dyno_panel.py

```diff
@@ -222,16 +222,28 @@ def writeFile():
 throttle_canvas.create_window(75, 45, window=throttle_slider)
 #---------------------
 
+def check_ack():
+    if not mcu_acknowledged:
+        data = driver.read()
+        if data:
+            mcu_acknowledged = True
+            print("ACK=SUCCESS;")
+        
+def mcu_flag(event):
+    mcu_acknowledged = False
+       
+#---------------------
 #Create the push button
 push_button = tk.Button(bg = color_bg, activebackground = color_bg, fg=color_fg, text = "Push", font=font_magnet)
 push_button.place(relx = 0.75, rely = 0.89, anchor='center')
 push_button.bind("<Button-1>", push_config)
+push_button.bind("<Button-1>", mcu_flag)
```

Comment:
We can just add the mcu_acknowledged = false statement to the push_config function instead of defining a separate function and binding the button